pi5/config.go

```diff
@@ -55,15 +41,8 @@ type ConfigConverter = func(resource.Config, logging.Logger) (*LinuxBoardConfig,
 // BeagleBone or Jetson boards.
```

Comment:
This repo is about the pi5 only, but these comments are about Beaglebone and Jetson boards. The comments (and possibly the code they describe?) should be cleaned up eventually, but it doesn't have to be in this PR.